### PR TITLE
FIX - lineageOS token lockout

### PR DESCRIPTION
**Pull request by rksh**
this may not be the best solution, but if we can't access the token, I suppose we should clear it?

### CODE REVIEW

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/TokenAccess.java [hunk 1]
```diff
@@ -226,8 +226,16 @@ public static String getApiToken(SharedPreferences prefs) {
...
+                    InvalidKeyException | BadPaddingException | IllegalBlockSizeException |
+                    UnsupportedOperationException ex) {
                 MainActivity.error("[TOKEN] Failed to get API Token: ", ex);
+                /**
+                 * ALIBI: in the event of failure, we'll clear, force a re-auth
+                 */
+                clearApiToken(prefs);
+                final SharedPreferences.Editor editor = prefs.edit();
+                editor.remove(ListFragment.PREF_AUTHNAME);
+                editor.apply();
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/TokenAccess.java]: i think this is good behavior, as long as it lets us clear in this situation. no idea how to test that. one thing though, can we let the user know this happened? just a toast or something?

Comment (rksh) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/TokenAccess.java]: how about handling this via ObservationUploader below?

**Comment by bobzilladev** [comment]
wfm +1

**Comment by rksh** [comment]
please do not merge yet - trying to find a way to confirm that this is sufficient to fix.

**Comment by bobzilladev** [comment]
Will the aes change fix things? Or would you like to de-conflict this?

**Comment by rksh** [comment]
doesn't seem particularly useful compared to https://github.com/wiglenet/wigle-wifi-wardriving/pull/157 - closing, we can re-add later if we're having trouble after AES conversion.

**Pull request closed without merging**